draft-ietf-httpbis-resumable-upload.md

```diff
@@ -85,7 +85,7 @@ This document defines an optional mechanism for HTTP that enables resumable uplo
 The terms Byte Sequence, Item, String, Token, Integer, and Boolean are imported from
 {{!STRUCTURED-FIELDS=RFC8941}}.
 
-The terms client and server are from {{HTTP}}.
+The terms "representation", "representation data", "representation metadata", "content", "client" and "server" are from {{HTTP}}.
```

Comment:
Is there a preferred way of written the referenced terms? Here you quote them, but in the sentence above we capitalize them without quotes. Does it depend on how the referenced document writes the terms?